src/linear_algebra/general_linear_group.lean

```diff
@@ -220,4 +236,19 @@ end coe_fn_instance
 
 end general_linear_group
 
+variables {n : Type u} [decidable_eq n] [fintype n] {R : Type v} [linear_ordered_comm_ring R]
+
+@[simp]
+lemma GL_pos_det_eq_det (A : GL_pos n R) : (det A) = (A : GL n R).det :=rfl
+
+@[simp]
+lemma coe_to_GL_pos_ext (g : special_linear_group n R) (i j : n) :
+   (g : (GL_pos n R)) i j = g i j := rfl
+
+@[simp] lemma coe_one_GL_pos : ((1 : special_linear_group n R) : GL_pos n R) = 1 :=
+begin
+ext,
+refl,
+end
+
```

Comment:
This section is ignoring the comment above that says "-- this section should be last to ensure we do not use it in lemmas"

Reply:
Hmm yes I know, I broke the rules there. The problem is ti wont let me state 
```
lemma coe_to_GL_pos_ext (g : special_linear_group n R) (i j : n) :
   (g : (GL_pos n R)) i j = g i j :=
```
before this. Do I need to prove an ext result for `GL_pos` first? Since at the moment it seems to want to make into a function first.

Reply:
That's not really an `ext` lemma in the first place - you could spell it
```
example (g : special_linear_group n R) :
  (↑(↑(↑(g : special_linear_group n R) : GL_pos n R) : GL n R) : matrix n n R) i j = (↑g : matrix n n R) := rfl
```
but my claim is that this lemma is useless, it's just a weaker version of the `coe_GL_pos_coe_GL_coe_matrix` we have above.